docs/static/azure-module.asciidoc

```diff
@@ -0,0 +1,591 @@
+bin/logstash
+-----
+
+[[exploring-data-azure]]
+==== Explore your data
+When the Logstash Azure module starts receiving events, you can begin using the
+packaged Kibana dashboards to explore and visualize your data. 
+
+To explore your data with Kibana:
+
+. Open a browser to http://localhost:5601[http://localhost:5601] (username:
+  "elastic"; password: "{pwd}")
+. Click *Dashboard*.
+. Select the dashboard you want to see.
+
+
+==== Azure module schema
+
+This module reads data from the Azure Event Hub and adds some additional structure to the data for Activity Logs and SQL Diagnostics. The original data is always preserved and any data added or parsed will be namespaced under 'azure'. For example, 'azure.subscription' may have been parsed from a longer more complex URN.
+
+[cols="<,<,<",options="header",]
+|=======================================================================
+|Name |Description|Notes
+
+|azure.subscription 	|Azure subscription from which this data originates. 	|Some Activity Log events may not be associated with a subscription.
+|azure.group 	|Primary type of data. |Current values are either 'activity_log' or 'sql_diagnostics' 	
+|azure.category* 	|Secondary type of data specific to group from which the data originated 	|
+|azure.provider 	|Azure provider 	| 
+|azure.resource_group 	|Azure resource group 	| 
+|azure.resource_type 	|Azure resource type 	| 
+|azure.resource_name 	|Azure resource name 	| 
+|azure.database 	|Azure database name, for display purposes 	|SQL Diagnostics only
+|azure.db_unique_id 	|Azure database name that is garunteed to be unique 	|SQL Diagnostics only
+|azure.server 	|Azure server for the database 	|SQL Diagnostics only
+|azure.server_and_database 	|Azure server and database combined 	|SQL Diagnostics only
+|azure.metadata 	|Any @metadata added by the plugins, for example var.input.azure_event_hubs.decorate_events: true 	
+|=======================================================================
+
+Notes:
+
+* Activity Logs can have the following categories: "Administrative", "ServiceHealth", "Alert", "Autoscale", "Security"
+* SQL Diagnostics can have the following categories: "Metric", "Blocks", "Errors", "Timeouts", "QueryStoreRuntimeStatistics", "QueryStoreWaitStatistics", "DatabaseWaitStatistics", "SQLInsights"
+
+Microsoft documents Activity log schema
+https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-activity-log-schema[here].
+The SQL Diagnostics data is documented
+https://docs.microsoft.com/en-us/azure/sql-database/sql-database-metrics-diag-logging[here].
+Elastic does not own these data models, and as such, cannot make any
+assurances of information accuracy or passivity.
+
+===== Special note - Properties field
+
+Many of the logs contain a "properties" top level field. This is often where the
+most interesting data lives. There is not a fixed schema between log types for
+properties fields coming from different sources. This can cause mapping errors
```

Comment:
> This can cause mapping errors ...

Can we nuke that sentence ? We cover it later with "To avoid mapping errors, ..."